CPAN/buildme.sh

```diff
@@ -435,8 +435,10 @@ if [ "$PERL_BIN" = "" -o "$CUSTOM_PERL" != "" ]; then
 
 fi
 
```

Comment:
This is an unrelated change and doesn't belong here. Please submit its own change request with information about why you do this change.

Reply:
@mherger This branch is reset.